src/Symfony/Component/Messenger/Transport/Doctrine/Exception/ConnectionNotFoundException.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Doctrine\Exception;
+
+use Symfony\Component\Messenger\Exception\InvalidArgumentException;
+
+/**
+ * @author Vincent Touzet <vincent.touzet@gmail.com>
+ */
+class ConnectionNotFoundException extends InvalidArgumentException
```

Comment:
Hmm, in general, I'm not sure if we need this class. It doesn't seem to me like a situation where a user would legitimately need to be trying to "catch" this specific error - it would be a configuration error.

Reply:
Maybe we can throw a `Symfony\Component\Messenger\Exception\TransportException` or a `Symfony\Component\Messenger\Exception\RuntimeException` instead of this ? :thinking:

Reply:
Go for using TransportException ^^